Documentation/project-docs/profiling-api-status.md

```diff
@@ -43,4 +43,26 @@ The notes below will help you determine what profiling APIs are safe to use. The
 ##APIs definitely known not to work yet
 ###Windows
 ###Linux
+
+* ICorProfilerInfo:
```

Comment:
Maybe adding the handle against the API in parens? Not the best idea but may work.